src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/PublishingCompletionService.cs

```diff
@@ -88,20 +91,27 @@ await releaseIdsToUpdate
             .Where(p => publicationSlugs.Contains(p.Slug))
             .Select(p => p.Id)
             .ToListAsync();
-        
+
         await directlyRelatedPublicationIds
             .ToAsyncEnumerable()
             .ForEachAwaitAsync(_publicationRepository.UpdateLatestPublishedRelease);
-        
+
+        // Set the published date on any methodologies used by these publications that are now publicly accessible
+        // as a result of releases being published
+        await directlyRelatedPublicationIds
+            .ToAsyncEnumerable()
+            .ForEachAwaitAsync(publicationId =>
+                _methodologyService.SetPublishedDatesByPublication(publicationId));
```

Comment:
I've changed it to `SetPublishedDatesIfApplicable` now, thanks.

I thought about other names but decided to go with `IfApplicable` to leave that down to the reader find out what makes it applicable or not, otherwise the name is going to be super long.